docker/ci/scripts/ui_tests.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# Script for running ui tests within a docker container.
+# In most cases, you will not run this script directly, but instead use
+# docker-compose to run using the ui-tests-compose.yml file in this directory.
+# See instructions in that file.
```

Comment:
It looks like .drone.yml does run these files directly. Could that be mentioned here? TBH I'm not totally clear on when we do use the compose.yml files, but my main request here is to provide accurate guidance for folks who are just trying to figure out what is / isn't getting executed as part of a drone run today.

Reply:
I can only guess at the original thinking behind this comment, but I think it's targeted at people who want to use them for local development; that's what the compose.yml files are for. I'll add some language targeted at people who want to build an understanding of what Drone does

Reply:
Thank you, looks great! Apologies, I didn't notice this was an existing comment.

Reply:
Always a problem when moving around code files :upside_down_face: